csharp/Squidex.ClientLibrary/Squidex.ClientLibrary/ContentsClient.cs

```diff
@@ -111,7 +111,21 @@ public Task<TEntity> UpdateAsync(Guid id, TData data, bool asDraft = false, Canc
             Guard.NotEmpty(id, nameof(id));
             Guard.NotNull(data, nameof(data));
 
-            return RequestJsonAsync<TEntity>(HttpMethod.Put, BuildSchemaUrl($"{id}/?asDraft={asDraft}", false), data.ToContent(), ct: ct);
+            if (asDraft)
+            {
+                QueryContext ctx = QueryContext.Default.Unpublished(true);
```

Comment:
The draft endpoint does not accept new content: https://github.com/Squidex/squidex/blob/master/backend/src/Squidex/Areas/Api/Controllers/Contents/ContentsController.cs#L478

You have to invoke POST .../draft first, then UPDATE /...

But I think it would be better to map the API methods 1:1 and then build "usability methods" on top of that as extension methods.

Furthermore I think it should be doable without the extra GET, I guess the API just fails (if you create a new draft) and then you can ignore the exception or so.

Reply:
I will verify, but it looks like when you call CreateDraftAsync when you already have a draft, it creates another draft.